projects/Mallard/src/screens/issue-screen.tsx

```diff
@@ -306,16 +310,20 @@ const IssueFronts = ({
             data={frontWithCards}
             style={style}
             key={width}
-            renderItem={({ item: front }) => (
-                <Front
-                    localIssueId={issue.localId}
-                    publishedIssueId={issue.publishedId}
-                    articleNavigator={frontSpecs}
-                    frontData={front}
-                    cards={front.cards}
-                    key={front.key}
-                />
-            )}
+            renderItem={({ item: front }) => {
+                return front ? (
+                    <Front
+                        localIssueId={issue.localId}
+                        publishedIssueId={issue.publishedId}
+                        articleNavigator={frontSpecs}
+                        frontData={front}
+                        cards={front.cards}
+                        key={front.key}
+                    />
+                ) : (
+                    <Text>hi</Text>
```

Comment:
@philmcmahon can't remember I added this here, not sure where it came from, very strange!